content/en/about/supported-releases/index.md

```diff
@@ -5,19 +5,66 @@ weight: 35
 icon: cadence
 ---
 
-This page lists the status and timeline for currently supported releases. Supported releases of Istio include releases that are in the active
+This page lists the status, timeline and policy for currently supported releases. Supported releases of Istio include releases that are in the active
 maintenance window and are patched for security and bug fixes. Subsequent patch releases on a LTS release do not contain backward incompatible
-changes. For more information refer to the [Istio support policy](../release-cadence/).
+changes.
+
+* [Support Policy](#support-policy)
+* [Naming scheme](#naming-scheme)
+* [Support status of Istio releases](#support-status-of-istio-releases)
+* [Releases without known Common Vulnerabilities and Exposures (CVEs)](#releases-without-known-Common-Vulnerabilities-and Exposures) 
+
+## Support policy
+
+We produce new builds of Istio for each commit. Around once a quarter or so, we build a Long Term Support (LTS) release,
```

Comment:
```suggestion
We produce new builds of Istio for each commit. Around once a quarter, we build a Long Term Support (LTS) release and run through several additional tests as well as release qualification. We issue patch releases for issues found in LTS releases. 
```

Reply:
I think the "LTS" is a misleading term, since it implies there is some short term supported releases, when there is not. I would just say something like "we build a release" or "we build an official release" or similar

Reply:
Agree. I would really like to see us move away from the term LTS as it causes a lot of confusion for users who expect LTS to mean something similar to Ubuntu and others who use the term. However, I think whether we call it LTS or not is beyond this PR.

Reply:
Removed LTS in favor of "minor" release. We can get more guidance from the TOC in this PR.

Reply:
That would be awesome. Thanks!